index.html

```diff
@@ -274,7 +274,7 @@ <h3>Double-sided Ruby, background</h3>
           A sequence of characters can be accompanied by two ruby annotations,
 	  typically consisting of [=Furigana=] and either [=GIKUN=] or an [=interlinear note=].  
           In <a data-cite="JLREQ#fig2_3_12">an example provided in JLreq</a> 
-          ("An example of ruby attached to both sides of the base characters"), 
+          ("An example of ruby annotations attached to both sides of the base characters"), 
```

Comment:
not the target of PR, but base characters or base text (or ruby base - was not the target, I believe)?